host/xtest/pkcs11_1000.c

```diff
@@ -716,15 +737,108 @@ static CK_RV test_already_initialized_token(ADBG_Case_t *c, CK_SLOT_ID slot)
 	return rv;
 }
 
-static CK_RV test_uninitialized_token(ADBG_Case_t *c, CK_SLOT_ID slot)
+static CK_UTF8CHAR test_acl_auth_token_label[] = "PKCS11 TA test token for ACL based authentication";
+
+static CK_RV init_test_token_acl_auth(CK_SLOT_ID slot)
+{
+	return C_InitToken(slot, NULL, 0, test_acl_auth_token_label);
+}
+
+#define TEE_UUID_NS_NAME_SIZE  128
+#define ACL_PIN_LEN  43
+
+/*
+ * TEE Client UUID name space identifier (UUIDv4)
+ *
+ * Value here is random UUID that is allocated as name space identifier for
+ * forming Client UUID's for TEE environment using UUIDv5 scheme.
+ */
+static const char *client_uuid_linux_ns = "58ac9ca0-2086-4683-a1b8-ec4bc08e01b6";
```

Comment:
I find the mention of randomness and UUIDv4 confusing (could this value be changed to another random UUID without breaking anything? Of course not). It relates to how the Linux UUID was chosen, but now it is irrelevant, what matters is that the proper UUID from the Linux kernel has to be used here as a namespace for UUIDv5 derivation. I suggest:
```
/*
 * TEE client UUID name space identifier
 * Client UUIDs are derived from this UUID (which is defined in the Linux
 * kernel) using the UUIDv5 scheme.
 */
```